osgeo_importer/templates/osgeo_importer/new.html

```diff
@@ -48,9 +48,6 @@ <h2 class="page-title">{% trans "Add data" %}</h2>
                 <i class="fa fa-warning">  {{ error|escape }}</i>
             {% endfor %}
             </div>
-            <div class="col-md-6" style="padding-top: 5px; padding-left: 2px;">
```

Comment:
Is there not need for this information? We could generate it using values defined in python if we wanted.